redisinsight/api/src/utils/logsFormatter.spec.ts

```diff
@@ -0,0 +1,177 @@
+import { BadRequestException, NotFoundException } from '@nestjs/common';
+import { CloudOauthMisconfigurationException } from 'src/modules/cloud/auth/exceptions';
+import { AxiosError, AxiosHeaders } from 'axios';
+import { mockSessionMetadata } from 'src/__mocks__';
+import { getOriginalErrorCause, sanitizeError, sanitizeErrors } from './logsFormatter';
+
+const error1 = new Error('Original error');
+error1['some'] = 'field';
+const error2 = new NotFoundException('Not found', { cause: error1 });
+const error3 = new BadRequestException('Bad req', { cause: error2 });
+const error4 = new CloudOauthMisconfigurationException('Misconfigured', { cause: error3 });
```

Comment:
These names are not very descriptive 
```suggestion
const genericError = new Error('Original error');
genericError['some'] = 'field';
const notFoundError = new NotFoundException('Not found', { cause: error1 });
const badRequestError = new BadRequestException('Bad req', { cause: error2 });
const cloudError = new CloudOauthMisconfigurationException('Misconfigured', { cause: error3 });
```

Reply:
Ok. I've changed them to more meaningful names